blocks.js

```diff
@@ -2518,6 +2547,14 @@ BlockMorph.prototype.userMenu = function () {
         'userDestroy'
     );
     menu.addItem(
+      'stringify...',
+        function() {
+            var code = myself.stringify();
+            window.prompt('scratchblocks code for you to copy and paste', code);
```

Comment:
That seems like a lot of code just for copy/pasting scratchblocks!

If you don't mind I'm not going to do that here, and instead open a new issue for that improvement. :)